frontend/public/components/cluster-settings/cluster-settings.tsx

```diff
@@ -961,10 +968,17 @@ export const ClusterOperatorTabPage: React.FC<ClusterOperatorTabPageProps> = ({
 export const ClusterSettingsPage: React.FC<ClusterSettingsPageProps> = ({ match }) => {
   const { t } = useTranslation();
   const title = t('cluster-settings~Cluster Settings');
-  const resources = [
+  const resources: FirehoseResource[] = [
     { kind: clusterVersionReference, name: 'version', isList: false, prop: 'obj' },
-    { kind: clusterAutoscalerReference, isList: true, prop: 'autoscalers', optional: true },
   ];
+  if (useFlag(FLAGS.CLUSTER_AUTOSCALER)) {
```

Comment:
I'd pull `useFlag` onto its own line outside the `if` since hooks always need to run in the same order every render. It's not a problem now since the `if` will always evaluate, but if someone added another condition later like `isOpenShift && useFlag(FLAGS.CLUSTER_AUTOSCALER)`, it could break things. It's a good practice to have hooks on their own near the top of the function.

Reply:
LOL.  I had it on its own line and then optimized.  That'll learn me.